### PR TITLE
Update the cw options a bit

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

Reduce IMEF on the "more" efficient option. Change drum volume on "most" efficient option. Add links in unit tests to actual clothes washer examples pulled from energystar.

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] PAT project measures/outputs have been updated
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing (green) on circleci
- [ ] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: resources/options_lookup.tsv [hunk 1]
```diff
@@ -1683,12 +1683,12 @@ Clothes Washer	"EnergyStar, 80% Usage"	ResidentialApplianceClothesWasher	imef=2.
...
+Clothes Washer	"EnergyStar Most Efficient, 80% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=75.0	annual_cost=7.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=0.8	mult_hw=0.8	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar Most Efficient, 100% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=75.0	annual_cost=7.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.0	mult_hw=1.0	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar Most Efficient, 120% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=75.0	annual_cost=7.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.2	mult_hw=1.2	location=auto	plant_loop=auto
```
Comment (ejhw) [inline resources/options_lookup.tsv]: @shorowit How would you feel about adding comments to options_lookup.tsv to document source of option argument values? e.g., 
># https://www.lowes.com/pd/Samsung-AddWash-4-5-cu-ft-High-Efficiency-Stackable-Front-Load-Washer-White-ENERGY-STAR/1000041269

Any better ways to do that?

**Comment by joseph-robertson** [comment]
@ejhw Should we merge this now, and worry about where this documentation should ultimately go later?

File: resources/options_lookup.tsv (hunk 1 above)
Comment (JLReyna) [reply resources/options_lookup.tsv]: I think the manufacturer and model number is the most appropriate way to do it. You can easily find the specs with those two things, while a link to a store might change.

I think this is a larger issue than just these options though. Given all of the confusion on the appropriate way to create new options for hot water-consuming appliances, I think we need to document all of our options. I vote we go ahead and merge and then deal with the documentation.

File: resources/measures/ResidentialApplianceClothesWasher/tests/ResidentialClothesWasher_Test.rb [hunk 2]
```diff
@@ -40,28 +40,30 @@ def test_new_construction_energystar
...
     args_hash["test_date"] = 2013
     expected_num_del_objects = {}
     expected_num_new_objects = { "ElectricEquipmentDefinition" => 1, "ElectricEquipment" => 1, "WaterUseEquipmentDefinition" => 1, "WaterUseEquipment" => 1, "ScheduleRuleset" => 1, "ScheduleConstant" => 1 }
-    expected_values = { "Annual_kwh" => 33.2, "HotWater_gpd" => 0.98, "Location" => args_hash["location"] }
+    expected_values = { "Annual_kwh" => 33.2, "HotWater_gpd" => 0.93, "Location" => args_hash["location"] }
     _test_measure("SFD_2000sqft_2story_FB_GRG_UA_3Beds_2Baths_Denver_WHTank.osm", args_hash, expected_num_del_objects, expected_num_new_objects, expected_values)
   end
 
   def test_new_construction_energystar_most_efficient
+    # https://www.lowes.com/pd/Samsung-AddWash-4-5-cu-ft-High-Efficiency-Stackable-Front-Load-Washer-White-ENERGY-STAR/1000041269
```
Comment (JLReyna) [inline resources/measures/ResidentialApplianceClothesWasher/tests/ResidentialClothesWasher_Test.rb]: I think this is fine for now. Recommend also including make / model number.

**Pull request merged**